evap/evaluation/migrations/0011_remove_textanswer_checked_and_hidden.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('evaluation', '0010_fill_textanswer_state'),
+    ]
+
+    operations = [
+    	migrations.RemoveField(
+            model_name='textanswer',
+            name='checked',
+        ),
+        migrations.RemoveField(
+            model_name='textanswer',
+            name='hidden',
+        ),
+    ]
```

Comment:
https://docs.djangoproject.com/en/1.7/topics/migrations/#data-migrations says
```
Migrations that alter data are usually called “data migrations”; they’re best written as separate migrations, sitting alongside your schema migrations.
```
So let's keep them and optimize them when we do the next general migration optimization.